contrib/test/integration/swap.yml

```diff
@@ -0,0 +1,24 @@
+---
+
+- name: Obtain current state of swap
+  command: swapon --noheadings --show NAME
```

Comment:
Missing *=* sign: I think you mean `--show=NAME`

(It doesn't matter, technically, because all you do is count lines, but it's still worth fixing).